src/qtiItem/helper/simpleParser.js

```diff
@@ -102,41 +122,68 @@ function buildTooltip(targetHtml, contentId, contentHtml) {
     };
 }
 
+function buildTable($elt, elt, options) {
+    elt.body = {
+        body: '',
+        elements: {}
+    };
+
+    const $parsedTable = parseContainer($elt, options);
+    elt.body.body = $parsedTable.body;
+    elt.body.elements = $parsedTable.elements;
+    return elt;
+}
+
 function parseContainer($container, options) {
-    var ret = {
+    let ret = {
         serial: util.buildSerial('_container_'),
         body: '',
         elements: {}
     };
 
-    _.each(_parsableElements, function(qtiClass) {
-        $container.find(qtiClass).each(function() {
-            var $qtiElement = $(this);
-            var element = buildElement($qtiElement, options);
+    $container.find('table').each(function () {
+        let $qtiElement = $(this);
+        let element = buildElement($qtiElement, options);
 
-            ret.elements[element.serial] = element;
-            $qtiElement.replaceWith(_placeholder(element));
-        });
+        element = buildTable($qtiElement, element, options);
+        ret.elements[element.serial] = element;
+        $qtiElement.replaceWith(_placeholder(element));
     });
 
-    $container.find(_getElementSelector('math', options.ns.math)).each(function() {
-        var $qtiElement = $(this);
-        var element = buildMath($qtiElement, options);
+    $container.find(_getElementSelector('img', '')).each(function () {
```

Comment:
please, use `_.each(_parsableElements, function(qtiClass) {` as it was previously
you skipped `printedVariable` and this is a reason why unit test failed

Reply:
`ret.elements[element.serial] = element;`
some way accidentally deleted this line , this was causing the test case failed